tests/phpunit/tests/image/editor.php

```diff
@@ -116,17 +116,26 @@ public function test_set_quality() {
 	 * @ticket 6821
 	 */
 	public function test_set_quality_with_image_conversion() {
+		remove_filter( 'image_editor_output_format', 'wp_default_image_output_mapping' );
```

Comment:
This should be removed as it's now no longer in core.

Reply:
`image_editor_output_format` is still in core (https://github.com/WordPress/wordpress-develop/blob/d013b20c323c820e930351b89c93e8bcbccf8089/src/wp-includes/class-wp-image-editor.php#L378) it is applied to the output format since 5.8. I believe removing this is required to make this test pass but I will double check

Reply:
This change isn't needed, removing.